bin/common.ml

```diff
@@ -134,6 +134,18 @@ let term =
          & flag
          & info ["debug-backtraces"] ~docs
              ~doc:{|Always print exception backtraces.|})
+  and+ build_logs =
+    let arg = Arg.(conv
+           ((fun s -> Result.map_error (Dune.Config.Build_logs.of_string s)
+                        ~f: (fun s -> `Msg s)),
+            fun pp x ->
+              Format.pp_print_string pp (Dune.Config.Build_logs.to_string x)))
+    in Arg.(value
+            & opt (some arg) None
+            & info ["build-logs"] ~docs ~docv:"BUILD_LOGS"
```

Comment:
What is this "BUILD_LOGS" string?  Maybe this should be `[preserve | clear-on-rebuild]`?

Reply:
It's the string that's displayed in the doc. I.e. when you do `dune build --help` you'll see `--build-logs=BUILD_LOGS`. You can use `MODE` here.